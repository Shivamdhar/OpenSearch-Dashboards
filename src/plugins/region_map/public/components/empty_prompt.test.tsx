/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */

import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import EmptyPrompt from './empty_prompt';

describe('empty_prompt', () => {
  describe('rendering', () => {
    it('renders the default Empty Prompt', () => {
      const wrapper = shallow(
        <EmptyPrompt iconType="test" title="test" bodyFragment="test" actions="" />
      );
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
