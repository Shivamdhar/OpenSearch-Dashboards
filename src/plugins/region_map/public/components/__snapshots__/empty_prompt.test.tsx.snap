// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty_prompt rendering renders the default Empty Prompt 1`] = `
<EuiEmptyPrompt
  actions=""
  body={
    <React.Fragment>
      <p>
        test
      </p>
    </React.Fragment>
  }
  iconType="test"
  title={
    <h2>
      test
    </h2>
  }
/>
`;
